fcrepo-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -34,6 +34,16 @@
 
 	</servlet-mapping>
 
+  <filter>
+    <filter-name>ETagFilter</filter-name>
+    <filter-class>org.springframework.web.filter.ShallowEtagHeaderFilter</filter-class>
```

Comment:
@acoburn, which is fine, but then why is it worth the effort of supporting ETags at all if "this really only affects the HTML UI, and as these resources are small"?